src/other/Table.js

```diff
@@ -219,23 +272,190 @@
             .remove()
         ;
         this._paginator.render();
-    };
+        
+        function fixedLabels(){
+            var divcol = context._parentElement.selectAll(".cols-wrapper").data([0]);
+            divcol
+                .enter()
+                .append("div")
+                .attr("class","cols-wrapper")
+            ;
+            divcol.selectAll(".labels-wrapper").data([0])
+                .enter()
+                .append("table")
+                .classed("labels-wrapper", true)
+            ;
+            divcol.exit().remove();
+
+            var divrow = context._parentElement.selectAll(".rows-wrapper").data([0]);
+            divrow
+                .enter()
+                .append("div")
+                .attr("class","rows-wrapper")
+            ;
+            divrow.selectAll(".labels-wrapper").data([0])
+                .enter()
+                .append("table")
+                .classed("labels-wrapper", true)
+            ;
+            divrow.exit().remove();
+
+            var rowsWrapper = context._parentElement.select(".rows-wrapper");
+            var colsWrapper = context._parentElement.select(".cols-wrapper");
+            var colLabelsWrapper = colsWrapper.select('.labels-wrapper');
```

Comment:
It turns out divcol = colsWrapper and divrow = rowsWrapper.  (it did not start that way) I will combine these.